flipperdevices/flipperzero-firmware
# [FL-3171] Introduce stealth mode and auto-selective lock

# What's new

- The "lock" menu option now locks with pin if it is set
- The "lock with pin" option has been replaced with the stealth mode, which disables most sounds and vibrations (direct hardware access is not affected, i.e. the music player)

# Verification 

- Lock with and without the pin
- Enable stealth mode, check that there are no notification sounds and that the status bar icon is present
- Go into the notification settings, check that the volume and vibro options are disabled and the word "stealth" is shown
- Disable the stealth mode, check that the sounds are back and that there's no more status bar icon

# Checklist (For Reviewer)
